src/Commands/Workflow/InfoCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Workflow;
+
+use Robo\Robo;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+class InfoCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Show all details for a workflow including operations.
+     *
+     * @command workflow:info
+     *
+     * @param string $site_id Site name that the workflow is part of.
+     *
+     * @option string $workflow-id ID of the workflow to show.
+     * @option string $latest-with-logs Just show the latest workflow with logs.
+     *
+     * @usage terminus workflow:info <site_name> <wf-id>
+     *   Show the workflow with id <wf-id> found on site <site_name>.
+     */
+    public function info($site_id, $options = ['latest-with-logs' => false, 'workflow-id' => '']) {
+        $container = Robo::getContainer();
+        $container->get('application')->find('workflow:info:status')->run($this->getInput(), $this->getOutput());
+        $container->get('application')->find('workflow:info:operations')->run($this->getInput(), $this->getOutput());
+        $container->get('application')->find('workflow:info:logs')->run($this->getInput(), $this->getOutput());
```

Comment:
Yup, it does work. Thanks for the feedback, I'll implement the base class as I agree that's probably better.